decentraland/unity-explorer
# fix: splash screen stuck

## What does this PR change?

Fixes #757 

Rework on how the splash animation is waited until it ends before transitioning into the splash screen.
Handled exceptions with the profile fetching at the authentication screen, which might block the flow.

## How to test the changes?

1. Launch the explorer
2. Observe that the splash screen animation works as expected
3. Check that the flow continues as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
